maven-parent/pom.xml

```diff
@@ -323,6 +345,22 @@
               </target>
             </configuration>
           </execution>
+
+          <execution>
+            <!-- Generate Ozone Distro Info -->
+            <id>Generate Ozone Distro Info</id>
+            <phase>generate-resources</phase>
+            <goals>
+              <goal>run</goal>
+            </goals>
+            <configuration>
+              <target>
+                <echo file="${project.build.directory}/ozone-info.txt">
```

Comment:
You mean the format of the file itself? I guess that JSON is always better, as it will allow to nest version infos like addons explicitly added for every app, whenever applicable.

If it's indeed the file that starts stating the versions of the Ozone apps... then `ozone-versions`?

Reply:
JSON can be good format for representing structured & hierarchical data, but not ideal for informational text intended for human readability, however for this use case should be fine I guess since we can process the JSON further to a generate a rich & display friendly format e.g HTML or Markdown.

Reply:
Let's call this `ozone-info.json` for now. Let's see where that leads us.